.shellcheckrc

```diff
@@ -1,2 +1,5 @@
 # Enable all optional checks.
 enable=all
+
+# Follow  source  statements even when the file is not specified as input.
```

Comment:
Two double spaces.

Reply:
Nice catch, thanks. I'll fix it in the follow-up.